README.md

```diff
@@ -49,14 +56,13 @@ This will run the Go unit tests. It relies on `pact` being available on your
 provide a stub service for the Cypress tests.
 
 ```
-docker-compose -f docker/docker-compose.cypress.yml up -d --build
+docker-compose -f docker/docker-compose.cypress.yml up -d --build --force-recreate
```

Comment:
Is `--force-recreate` necessary? If you need to restart containers then it is better to use `down` then `up`

Reply:
done